setup.py

```diff
@@ -11,11 +11,9 @@
 
 setup(
     name="pylint-pytest",
-    version="1.1.2",
-    author="Reverb Chu",
-    author_email="pylint-pytest@reverbc.tw",
-    maintainer="Reverb Chu",
-    maintainer_email="pylint-pytest@reverbc.tw",
+    version="1.1.3a0",
+    author="Stavros Ntentos",
+    author_email="133706+stdedos@users.noreply.github.com",
```

Comment:
This is obviously fake: If it "somehow helps" we can keep it.

Reply:
That's the anonymized email github give you so it can credit you in commit but not show your actual email. It says "contact me on github only, punk"